README.md

```diff
@@ -1,3 +1,11 @@
 # data-eden
 
 TODO: writeme
+
+## Contributing
+
+To run the quality hooks before you commit your code please run the following to install the hooks
+
+```bash
+printf '#!'"/bin/sh\n\nnpx lint-staged\n" > .git/hooks/pre-commit
```

Comment:
Thank you for figuring this out, I forgot that you also needed to add the shebang

Reply:
same I kept running into issues.